tests/acceptance/gkeep_publish.robot

```diff
@@ -0,0 +1,85 @@
+# Copyright 2018 Nathan Sommer and Ben Coleman
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+
+*** Settings ***
+Resource    resources/setup.robot
+Test Setup    Setup Server and Client Accounts
+Force Tags    gkeep_publish
+
+*** Keywords ***
+
+Setup Server and Client Accounts
+    Launch Gkeepd And Configure Admin Account on Client
+    Add Faculty and Configure Accounts on Client    faculty1
+    Establish Course  faculty1    cs1     student1  student2
+    Create Accounts On Client   student1    student2
+    Add Assignment to Client  faculty1  good_simple
+    Gkeep Upload Succeeds   faculty1   cs1    good_simple
+
+*** Test Cases ***
+Valid Assignment Publish
+    [Tags]  happy_path
+    Gkeep Publish Succeeds  faculty1    cs1     good_simple
+    Email Exists    student1    good_simple
+    Email Exists    student2    good_simple
+    Clone Assignment  student1  faculty1    cs1     good_simple
+    Clone Assignment  student2  faculty1    cs1     good_simple
+
+Bad Assignment Name In Publish
+    [Tags]  error
+    Gkeep Publish Fails     faculty1    cs1     unknown_name
+
+Bad Course Name In Publish
+    [Setup]     NONE
```

Comment:
By using `[Setup]  NONE` does that mean that `Setup Server and Client Accounts` is not run before this test? If that is the case, will the user `faculty1` not exist on the server, thus the publish command would fail because the faculty user does not exist rather than because the course does not exist?